gatsby-config.js

```diff
@@ -4,6 +4,19 @@ require("dotenv").config({
   path: `.env.${process.env.NODE_ENV}`,
 })
 
+const path = require('path');
+// Get paths of Gatsby's required rules, which as of writing is located at:
+// https://github.com/gatsbyjs/gatsby/tree/fbfe3f63dec23d279a27b54b4057dd611dce74bb/packages/
+// gatsby/src/utils/eslint-rules
+const gatsbyRequiredRules = path.join(
+  process.cwd(),
+  'node_modules',
+  'gatsby',
+  'dist',
+  'utils',
+  'eslint-rules'
+);
```

Comment:
Looks like this doesn't complain if that path does not exist, which can be problematic if they ever change the name upstream: we wouldn't notice anything.

Reply:
I just followed the steps listed in the official plugin page, so I assume they intend to keep this path as is for this version of the plugin